### PR TITLE
Add `timestamp()` Fix function.

### LINKED ISSUE #230: Add `timestamp`-function
At the moment there is no timestamp function like in [morph](https://github.com/metafacture/metafacture-core/blob/master/metamorph/src/main/java/org/metafacture/metamorph/functions/Timestamp.java).

I need this for the Alma-Transformation in Fix.

### PR DESCRIPTION
**Pull request by blackwinter**
Resolves #230.

### CODE REVIEW

File: metafix/src/main/java/org/metafacture/metafix/api/FixFunction.java [hunk 1]
```diff
@@ -24,13 +24,20 @@
...
 import java.util.Set;
+import java.util.function.Consumer;
 import java.util.stream.Stream;
 
 @FunctionalInterface
 public interface FixFunction {
 
     void apply(Metafix metafix, Record record, List<String> params, Map<String, String> options);
 
+    default void getOption(final Map<String, String> options, final String key, final Consumer<String> consumer) {
```
Comment (fsteeg) [inline metafix/src/main/java/org/metafacture/metafix/api/FixFunction.java:35]: Maybe something like `consumeOption` or `getAndConsumeOption` would be clearer? (`getOption` is fine with me though).

**Review by fsteeg — APPROVED** [review]

File: metafix/src/main/java/org/metafacture/metafix/api/FixFunction.java (hunk 1 above)
Comment (blackwinter) [reply metafix/src/main/java/org/metafacture/metafix/api/FixFunction.java:35]: Yeah, I wasn't really sure about it myself. Naming things is hard ;)

Originally, I had `withOption()`. But then I opted for alignment with the existing `getBoolean()`/`getInteger()` instead.

Comment (fsteeg) [reply metafix/src/main/java/org/metafacture/metafix/api/FixFunction.java:35]: > Originally, I had `withOption()`.

I like that, reads good: `withOption ... setFormat`

Comment (blackwinter) [reply metafix/src/main/java/org/metafacture/metafix/api/FixFunction.java]: Okay, I've changed the name in 710db71.

Comment (fsteeg) [reply metafix/src/main/java/org/metafacture/metafix/api/FixFunction.java]: :+1:

**Comment by blackwinter** [comment]
Yes, that's the default behaviour:

> By default it returns a unix timestamp.

— [org.metafacture.metamorph.functions.Timestamp](https://javadoc.io/doc/org.metafacture/metamorph/latest/org/metafacture/metamorph/functions/Timestamp.html)

**Comment by TobiasNx** [comment]
+1 I tested it and added documentation in the `README.md`

Thanks

**Comment by blackwinter** [comment]
Thanks for the documentation! I've tweaked it a little.

**Pull request merged**